src/main/java/frc/robot/commands/autos/OneFiveLeave.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.autos;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.IntakeSequence;
+import frc.robot.commands.autos.components.DriveDistance;
+import frc.robot.commands.autos.components.ScoreThisNote;
+import frc.robot.subsystems.conveyor.Conveyor;
+import frc.robot.subsystems.drive.DriveConstants;
+import frc.robot.subsystems.drive.DriveSubsystem;
+import frc.robot.subsystems.elevator.Elevator;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.intake.Intake.IntakePosition;
+import frc.robot.subsystems.shooter.Shooter;
+
+/** score the preloaded note into the speaker, then get another note, return to home state */
+public class OneFiveLeave extends SequentialCommandGroup {
+  public OneFiveLeave(Intake intake, Elevator elevator, Shooter shooter, Conveyor conveyor, DriveSubsystem drive) {
+    final double X_DISTANCE_TO_NOTE_METERS = 1.0; // guesstimate
+    final double NO_Y_DISTANCE = 0.0;
+    final boolean FIELD_RELATIVE = true;
+
+    addRequirements(intake, elevator, shooter, conveyor, drive);
+    addCommands(
+      new ScoreThisNote(intake, elevator, shooter, conveyor),
+      new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.DEPLOYED)),
+      new DriveDistance(drive, DriveConstants.SLOW_LINEAR_SPEED_METERS_PER_SEC, X_DISTANCE_TO_NOTE_METERS, NO_Y_DISTANCE, FIELD_RELATIVE),
+      new IntakeSequence(intake, elevator, conveyor)
```

Comment:
I think you need to be intaking while you drive

Reply:
Also consider what if intaking doesn't succeed. This will just keep running the intake and conveyor indefinitely. Maybe a timeout or something?